lib/honeybadger/config.js

```diff
@@ -0,0 +1,46 @@
+import Honeybadger from "@honeybadger-io/js";
+
+const setupHoneyBadger = () => {
+  // https://docs.honeybadger.io/lib/javascript/reference/configuration.html
+  const sharedHoneybadgerConfig = {
+    apiKey: process.env.HONEYBADGER_API_KEY,
+    environment: process.env.NODE_ENV,
```

Comment:
Recommending we change this to
```suggestion
    environment: process.env.HONEYBADGER_ENV || process.env.NODE_ENV,
```
so that we can override it if we want without monkeying with the `NODE_ENV` variable.